### ISSUE TITLE
OPEN DATA: Fisheries Dependent data for Sea Scallop in the Southern Gulf of St. Lawrence (NEW)

### ISSUE THREAD

**Issue by davjfish** [dfo-gulf-science/CADI#137]
Today @JenniMcD and I had a meeting with Andrew and Monique to discuss the feasibility of this record. The initial thought was that this should be easy to churn out. 

The record will consist of the corrected landings data from Stats and the corresponding indecies of fishing effort. 
Each row in the data should be the computed annual effort for a given region/SFA. In years where the breakdown by SFA is not available, an aggregated region can be used instead, e.g., 4T. 

Metadata should make sure to reference the related CSAS publication and should have a brief overview of the steps involved for the production of indices.

- [x] identify data steward
- [ ] data steward to complete metadata
- [ ] data steward to certify metadata records
- [ ] internal CADI review of metadata
- [ ] obtain dataset from data steward
- [ ] creation of map / dataset to be used for WMS (data steward and/or CADI)
- [ ] submit for level 1 review
- [ ] release criteria checklist
- [ ] submit for publication
- [ ] complete review of metadata record for compliancy

**Comment by landrylys** [comment]
Hi, I just updated the dataset (and dictionary) with the new data set used in the analysis done by Andrew. If it has a problem, do not hesitate. (New decisions have been made in the light of the CSAS.)

**Comment by RobichaudA** [comment]
On going issues with this file

- Dataset has been updated yet again, column headers are the same, so data dictionary will also need to be updated
- Will need to double check on geojson to see if any updates are required there.
- Metadata may need to be updated. Andrew mentioned yesterday that it was mentioned in the CSAS report that the 2023 data was preliminary, and he felt that this should be reflected somewhere. I advised him that I was leaving this up to him or Monique to do as I did not want to write something incorrectly in the metadata.
